lms/envs/aws.py

```diff
@@ -834,3 +834,5 @@
 APP_UPGRADE_CACHE_TIMEOUT = ENV_TOKENS.get('APP_UPGRADE_CACHE_TIMEOUT', APP_UPGRADE_CACHE_TIMEOUT)
 
 AFFILIATE_COOKIE_NAME = ENV_TOKENS.get('AFFILIATE_COOKIE_NAME', AFFILIATE_COOKIE_NAME)
+
+ENABLE_COMBINED_LOGIN_REGISTRATION_FOOTER = ENV_TOKENS.get('ENABLE_COMBINED_LOGIN_REGISTRATION_FOOTER',  FEATURES['ENABLE_COMBINED_LOGIN_REGISTRATION_FOOTER'])
```

Comment:
As is implemented, this flag `ENABLE_COMBINED_LOGIN_REGISTRATION_FOOTER` is a feature flag (a key in the `settings.FEATURES` dictionary) and not a top-level setting (defined directly on `settings`), so we don't want this line.

The entire `FEATURES` dict is already updated from JSON.